pkg/virt-launcher/monitor_test.go

```diff
@@ -114,6 +112,9 @@ var _ = Describe("VirtLauncher", func() {
 	}
 
 	BeforeEach(func() {
+		if !strings.Contains(fakeQEMUBinary, "../../") {
```

Comment:
Why this change was needed?

Reply:
https://github.com/kubevirt/kubevirt/pull/3945#discussion_r467520465

Reply:
How is this related to the flags we parse? Also, this approach seems wrong to me... if the fakeQEMUBinary doesn't exist exactly at the specified path, we should fail and not try to recover by prefixing the path with `../../`